Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -8,6 +11,13 @@ namespace
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    batteryData_.current = 0;
+    batteryData_.time.setHMS(0,0,0,0);
+    batteryData_.voltage = 0;
+    totalAmpHoursUsed_ = 0;
+    ampHours_ = (initialStateOfChargePercent_/100)*BATTERY_AMP_HOUR_CAPACITY;
+    dataPoints_ = 0;
+    timeDiff_ = 0;
```

Comment:
are these private variable defined in the header files? Also, ampHours_ doesn't quite seem to be logically correct, think about what it is actually means by the initalStateOfCharge.